C++ version is low in Azure pipelines Ubuntu

```
Scanning dependencies of target tvgw_static
[ 10%] Building CXX object CMakeFiles/tvgw_static.dir/script/cpp/src/tvgw_converter.cpp.o
/home/vsts/work/1/s/script/cpp/src/tvgw_converter.cpp: In function ‘TVGW_RIFF_File* tvgw_ConvertToRIFF(unsigned int, unsigned int, TVGW_File_Chunk**)’:
/home/vsts/work/1/s/script/cpp/src/tvgw_converter.cpp:25:10: error: ‘riff_contents’ does not name a type
     auto riff_contents = new TVGW_RIFF_File;
          ^
/home/vsts/work/1/s/script/cpp/src/tvgw_converter.cpp:34:5: error: ‘riff_contents’ was not declared in this scope
     riff_contents->RIFF = TVGW_RIFF;
     ^
CMakeFiles/tvgw_static.dir/build.make:62: recipe for target 'CMakeFiles/tvgw_static.dir/script/cpp/src/tvgw_converter.cpp.o' failed
make[2]: *** [CMakeFiles/tvgw_static.dir/script/cpp/src/tvgw_converter.cpp.o] Error 1
CMakeFiles/Makefile2:72: recipe for target 'CMakeFiles/tvgw_static.dir/all' failed
make[1]: *** [CMakeFiles/tvgw_static.dir/all] Error 2
Makefile:83: recipe for target 'all' failed
make: *** [all] Error 2
```
I think this error is caused because the reserved word, "auto" cannot be used in Ubuntu by C++ version.

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -2,6 +2,8 @@ cmake_minimum_required(VERSION 3.12.0)
 
 project(traveling-wav)
 enable_language(CXX)
+set(CMAKE_CXX_STANDARD 14)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
 
 set(CMAKE_CONFIGURATION_TYPES Debug Release)
 set(CMAKE_RUNTIME_OUTPUT_DIRECTORY_DEBUG "${CMAKE_BINARY_DIR}/Debug")

